graphql-kickstart-spring-boot-autoconfigure-graphql-annotations/src/main/java/graphql/kickstart/graphql/annotations/GraphQLAnnotationsAutoConfiguration.java

```diff
@@ -52,7 +53,14 @@ public GraphQLInterfaceTypeResolver graphQLInterfaceTypeResolver() {
   @Bean
   @ConditionalOnMissingBean
   public GraphQLAnnotations graphQLAnnotations() {
-    return new GraphQLAnnotations();
+    GraphQLAnnotations graphQLAnnotations = new GraphQLAnnotations();
+    if (nonNull(graphQLAnnotationsProperties.getInputPrefix())) {
```

Comment:
You could use an Optional instead, e.g.
```java
ofNullable(graphQLAnnotationsProperties.getInputPrefix()).ifPresent(graphqlAnnotations.getContainer()::setInputPrefix);
```

Reply:
Obviously both styles are being used in this project.

IMHO `ofNullable(something).ifPresent()` adds zero to the readability of this particular piece of code and it is not what Optional is intended for. It would be a different story if `GraphQLAnnotationsProperties#inputPrefix` already was an Optional. But that's not supported by Spring Boot.